2009-02-09  Michael Tautschnig  <mt@debian.org>

	* lib/setup-storage/Commands.pm: Added debug output in command dependency
		sorter
Index: trunk/lib/setup-storage/Commands.pm
===================================================================
--- trunk.orig/lib/setup-storage/Commands.pm
+++ trunk/lib/setup-storage/Commands.pm	
@@ -929,6 +929,11 @@
   my $pushed = -1;
 
   while ($i < $FAI::n_c_i) {
+    if ($FAI::debug) {
+      print "Trying to add CMD: " . $FAI::commands{$i}{cmd} . "\n";
+      defined($FAI::commands{$i}{pre}) and print "PRE: " .  $FAI::commands{$i}{pre} . "\n";
+      defined($FAI::commands{$i}{post}) and print "POST: " .  $FAI::commands{$i}{post} . "\n";
+    }
     my $all_matched = 1;
     if (defined($FAI::commands{$i}{pre})) {
       foreach (split(/,/, $FAI::commands{$i}{pre})) {
